cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller_test.go

```diff
@@ -825,64 +827,59 @@ func TestControllerNodeGroupForNodeLookup(t *testing.T) {
 			nodeGroupMaxSizeAnnotationKey: "2",
 		},
 		lookupSucceeds: true,
-	}} {
-		test := func(t *testing.T, i int, annotations map[string]string, useMachineDeployment bool) {
-			t.Helper()
+	}}
 
-			machineSet, machineDeployment := makeMachineOwner(i, 1, annotations, useMachineDeployment)
+	test := func(t *testing.T, tc testCase, testObjs *clusterTestConfig, node *apiv1.Node) {
+		t.Helper()
 
-			node, machine := makeLinkedNodeAndMachine(i, machineSet.Namespace, v1.OwnerReference{
```

Comment:
So basically this test does not need a machine to be linked with a node.

Reply:
> So basically this test does not need a machine to be linked with a node.

Node linkage is always done in `newMachineSetTestObjs()` now.

Reply:
Ok, inside `makeClusterObjs`. Got it!